src/test/regress/sql/citus_local_tables_queries.sql

```diff
@@ -1,3 +1,15 @@
+--
+-- CITUS_LOCAL_TABLES_QUERIES
+--
+-- Test queries on a distributed table with shards on the coordinator
+--
+-- This test file has an alternative output because of the change in the
+-- display of SQL-standard function's arguments in INSERT/SELECT in PG15.
+-- The alternative output can be deleted when we drop support for PG14
+--
+SHOW server_version \gset
+SELECT substring(:'server_version', '\d+')::int > 10 AS server_version_above_fourteen;
```

Comment:
```suggestion
SELECT substring(:'server_version', '\d+')::int > 14 AS server_version_above_fourteen;
```

I guess there is a copy paste error here. I wonder how that does not result in test failures.

Please go over similar changes in other files as well.

Reply:
Hi Hanefi, the PR is not ready for review